### PR TITLE
[Doc] Modify tensor_if README.md

### PR DESCRIPTION
**Pull request by gichan-jang**
- Remove description of unsupported properties.
 - Elaborate description.

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_if/README.md [hunk 1]
```diff
@@ -45,39 +37,26 @@ The input/output stream data type is either ```other/tensor``` or ```other/tenso
...
-
-- else_option: Option for FALSE Action
-  * nth tensor,nth tensor,... (for TENSORPICK option)
-  * ${path_to_file}
+  * PASSTHROUGH: Does not let you make changes to the buffers. Buffers are pushed straight through.
+  * SKIP: Does not let you generate the output frame (frame skip).
+  * TENSORPICK: Lets you choose the nth tensor among the input tensors.
+
+- else-option: Option for FALSE Action
+  * nth tensor: used for TENSORPICK option, for example, `then-option`=0,2 means tensor 0 and tensor 2 are selected as output tensors among the input tensors.
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_if/README.md:59]: Hmm... Is this for `else-option`?

Comment (gichan-jang) [reply gst/nnstreamer/tensor_if/README.md]: My bad. Updated.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**